connect/cli/plugins/shared/translation_attr_sync.py

```diff
@@ -103,6 +121,41 @@ def _update_attributes(self, translation_id, attributes, ws):
                 range(1, len(attributes) + 1),
             )
 
+    def _update_new_attrs_for_clone(self, new_translation_id, attributes, ws):
```

Comment:
apart from fixing the 10 items issue for bulk update, this method does the same than `_update_attributes`, can't use just that, with the fix applied?